test/raven.test.js

```diff
@@ -1732,8 +1746,9 @@ describe('Raven (public API)', function() {
         it('should not throw an error if not configured', function() {
             this.sinon.stub(Raven, 'isSetup').returns(false);
             this.sinon.stub(window, 'send')
-            Raven.captureMessage('foo');
-            assert.isFalse(window.send.called);
```

Comment:
Can we modify this test then to assert that `window.makeRequest` wasn't called? Since that's the changed behavior here.